charts/kyverno-policies/ci/test-autogen-none.yaml

```diff
@@ -0,0 +1,2 @@
+podSecurityStandard: restricted
```

Comment:
Will the CI picks up the test automatically?

Reply:
It should but I used wrong filename convention. As long as its `*-values.yaml` is's picked up automatically.

Fixed it

Reply:
I found this reference that works with the CircleCI:

>In order to enable custom test values, create a directory ci in the chart's directory and add any number of *-values.yaml files to this directory. 

We are using GitHub Actions, does this also apply?

Reply:
Looks like we already have other `*-values.yaml` tests defined. This test got picked up for the linter checks.

https://github.com/kyverno/kyverno/actions/runs/3590889725/jobs/6044795042